src/components/nav_item.js

```diff
@@ -4,7 +4,12 @@ import { browserHistory } from 'react-router';
 class NavItem extends Component {
   handleClick() {
     this.props.changeSelected(this.props.index);
-    browserHistory.push(this.props.link);
+    console.log(this.props.link)
+    if (this.props.link.substring(0, 1) !== '/') {
```

Comment:
What is this doing? I might be the only one who doesn't know, but a comment here describing what is happening would be helpful for me

Reply:
Good point, I'm just distinguishing the difference between a website link starting with `http` and a route in our project that starts with a `/`. Since two of our nav bar items are full website links, I had to make this check in order to change the `window.location` to the link instead of pushing with `browserHistory`. I'll add a comment though